tutorbackup/templates/backup/build/backup/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.8
+ENV PYTHONUNBUFFERED 1
+RUN curl -fsSL https://www.mongodb.org/static/pgp/server-4.2.asc | gpg --dearmor | tee /usr/share/keyrings/mongodb-ce-archive-keyring.gpg && \
+    echo "deb [arch=amd64,arm64 signed-by=/usr/share/keyrings/mongodb-ce-archive-keyring.gpg] https://repo.mongodb.org/apt/debian stretch/mongodb-org/4.2 main" | tee /etc/apt/sources.list.d/mongodb-org-4.2.list && \
+    apt-get update && \
+    apt-get install -y --no-install-recommends default-mysql-client mongodb-org-tools=4.2.17 && \
```

Comment:
Any particular reason why you want to nail this to the `.17` point release?

Reply:
Good catch! It was a copy paste from MongoDB docker image version number in Tutor. I'll remove that part.